dotnetcore3.1/cookiecutter-aws-sam-quick-start-from-scratch-dotnet/{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Information": [
+    "This file provides default values for the deployment wizard inside Visual Studio and the AWS Lambda commands added to the .NET Core CLI.",
+    "To learn more about the Lambda commands with the .NET Core CLI execute the following command at the command line in the project root directory.",
+    "dotnet lambda help",
+    "All the command line options for the Lambda command can be specified in this file."
+  ],
+  "profile": "",
+  "region": "",

```

Comment:
nit: Can we remove properties which have empty values?

Reply:
@mndeveci done!

Reply:
Thanks! But I was just referring to fields not the whole file. Because the file is already present in the current templates. https://github.com/aws/aws-sam-cli-app-templates/blob/master/dotnetcore3.1/cookiecutter-aws-sam-hello-dotnet/%7B%7Bcookiecutter.project_name%7D%7D/src/HelloWorld/aws-lambda-tools-defaults.json

Do you know what is the role of this file, can we just remove it and this will work without any issues?

Reply:
Discussed offline that this file is not required for application to build and/or deploy.